backend-app/controllers/users_controllers/admin_controller.ts

```diff
@@ -197,92 +137,117 @@ export const unbanUser = async (req: IReq, res: IRes, next: INext) => {
             throw new AppError(400, 'User is not banned');
         user.accessRestricted = false;
         await user.save();
-        res.status(200).json({
+        this.setStatus(200);
+        return {
             message: 'User is now unbanned',
-        });
-    } catch (err) {
-        next(err);
+        };
     }
-};
 
-export const createRole = async (req: IReq, res: IRes, next: INext) => {
-    const { name, authorities, restrictions } = req.body;
-    try {
+    @Response(400, 'Role already exists')
+    @SuccessResponse('201', 'CREATED')
+    @Middlewares(restrictTo(Actions.MANAGE_ROLES))
+    @Security('jwt')
```

Comment:
In this case, you don't have to add a security decorator to each request; just add it to the controller.

Reply:
okay.